Add scripts

mbtamuli opened the pull request:
- [x] Add install script
- [x] Add `eev4` global script skeleton
- [x] Add `eev4 create SITE --wp` Single Site command
- [x] Add `eev4 delete SITE` Delete Site command
- [x] Add `eev4 create SITE --wpsubdir` Multisite with Subdirectories command
- [x] Add `eev4 create SITE --wpsubdomain` Multisite with Subdomains command
- [x] Add `eev4 wp SITE [ARGS..]` wp-cli wrapper command

mbtamuli committed:
Add install script

Signed-off-by: Mriyam Tamuli <mbtamuli@gmail.com>

rahul286 commented on scripts/setup:
```diff
@@ -0,0 +1,91 @@
+PROJECT_NAME="eev4"
+
+# runs the given command as root (detects if we are root already)
+runAsRoot() {
```
Why we need root?

is it for /etc/hosts?

mbtamuli replied:
For that also we'll need root privileges, but that is not part of the install script.

We're installing the repo in `/opt/eev4` and symlinking `/opt/eev4/scripts/eev4` to `/usr/local/bin/eev4` to put it in PATH.
```
runAsRoot cp -r "$EE_TMP_REPO" "$EE_INSTALL_DIR"
runAsRoot ln -s "$EE_INSTALL_DIR/$EE_SCRIPT" "$EE_SCRIPT_INSTALL_PATH"
```

mbtamuli committed:
Add skeleton for eev4

Signed-off-by: Mriyam Tamuli <mbtamuli@gmail.com>

mbtamuli committed:
Add support for `eev4 create SITE --wp`

Signed-off-by: Mriyam Tamuli <mbtamuli@gmail.com>

mbtamuli committed:
Update scripts

1. Exit setup script if dependencies not installed
2. Add support for `eev4 delete SITE`

Signed-off-by: Mriyam Tamuli <mbtamuli@gmail.com>

mbtamuli committed:
Add support for `eev4 wp SITE [ARGS...]`

Signed-off-by: Mriyam Tamuli <mbtamuli@gmail.com>

mbtamuli commented:
After testing once, i am merging this. Looks okay to me. If problems arise, will fix those later.
